inference-engine/thirdparty/clDNN/tests/main.cpp

```diff
@@ -1,11 +0,0 @@
-// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
Why is it deleted?

Reply:
replaced with `gtest_main` which does the same

Reply:
I see, thanks.